tests/service/data/samename/tempet.fsproj

```diff
@@ -0,0 +1,13 @@
+<Project Sdk="FSharp.NET.Sdk;Microsoft.NET.Sdk">
```

Comment:
yes it's a NUnit test that loads the proj. I think it doesn't use the sdk at all